ui/src/search.tsx

```diff
@@ -0,0 +1,38 @@
+import SearchIcon from "@mui/icons-material/Search";
+import Box from "@mui/material/Box";
+import IconButton from "@mui/material/IconButton";
+import Stack from "@mui/material/Stack";
+import { TextField } from "mui-rff";
+import { Form } from "react-final-form";
+
+export type SearchProps = {
+  placeholder?: string;
+  onSearch: (query: string) => void;
+};
+
+export function Search(props: SearchProps) {
+  return (
+    <Box m={2}>
+      <Form
```

Comment:
Does the enter key work automatically with this pattern?

Reply:
Yeah, that's why I used the form.